.eslintrc.js

```diff
@@ -40,6 +40,8 @@ module.exports = {
     after: true,
     afterEach: true,
     URLSearchParams: true,
+    Worker: true,
+    self: true,
```

Comment:
Too broad setting. Placing a single comment in the beginning of `wasm.worker.js` should be enough